universe/com.goal_smart/device.js

```diff
@@ -0,0 +1,197 @@
+"use strict";
+
+const Tp = require('thingpedia');
+
+module.exports = class GoalDevice extends Tp.BaseDevice {
+  constructor(engine, state) {
+    super(engine, state);
+    this.uniqueId = 'com.goal_smart'; //
+    this.name = 'GoalSmart';
+    this.description = 'Keep up with the latest in soccer';
+  }
+
+  // getting league standings, given league id
+  get_standings({ league_id }) {
+    return Tp.Helpers.Http.get('https://api-football-v1.p.rapidapi.com/v3/standings?season=2020&league=' + league_id, {
+      extraHeaders: {
+        'x-rapidapi-key': this.constructor.metadata.auth.api_key,
+        'x-rapidapi-host': 'api-football-v1.p.rapidapi.com',
+        useQueryString: true
+      },
+      accept: 'application/json'
+    }).then((tempResponse) => {
+      const a = JSON.parse(tempResponse);
+      const b = a.response[0].league.standings[0];
+      return b.map((obj) => {
+        const r = obj.rank;
+        const n = obj.team.name;
+        const f = obj.form;
+        const p = obj.points;
```

Comment:
It's a very minor point, but I think the code would be more readable without the intermediate variable, becauseone has to connect the readable property name in the output to the property name in the `obj` through the variable name, and the variable has a very short name.